client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts

```diff
@@ -61,38 +61,38 @@ const useContactQuestioning = (parameters: useContactQuestioningParameters): use
             .finally(() => setIsLoading(false));
     };
 
-    const getRulerApiDataFromServer = async (ids : any []) => {
```

Comment:
can you please discard these changes it will cost conflicts and i prefer to fix these spaces after everything will work...